public/qos-restriction.txt

```diff
@@ -0,0 +1,26 @@
+request restricted due to resource limitations
+
+Resource limitations may vary from time to time depending on load and other conditions.
+You are unlikely to encounter a resource limitation if you use the data API for data accesss
+from programs and  issue at most two simultaneous data requests.  Data requests include 
+SPARQL queries, data API requests.
+
+Resource limitations for APIs intended to be used by the user interface are more restrictive
+than resource limitations on the data API and SPARQL queries.  If you are trying to retrieve
+data you are likely better off using the data API or SPARQL rather than scraping the UI pages
+or using UI specific APIs.
+
+See: 
+  http://landregistry.data.gov.uk/app/doc/ukhpi
+  http://landregistry.data.gov.uk/app/root/doc/ppd
```

Comment:
@der the links need the correct protocol in the uris:
- `https://landregistry.data.gov.uk/app/ukhpi/doc`
- `https://landregistry.data.gov.uk/app/root/doc/ppd`

Reply:
Fair enough, the sites all redirect to https but you are better to go there in the first place. Done.